cmd/charm/charmcmd/attach.go

```diff
@@ -35,6 +35,20 @@ using another channel.
 
     charm attach ~user/mycharm mydata=./blah -c unpublished
 
+The attach command can also be used to attach docker resources
+to Kubernetes charms. When a charm has a docker image resource,
+it can be attached by naming the image in the local docker instance:
+
+    charm attach ~user/mykubernetes-charm image=ubuntu
```

Comment:
I think the word image is a little overloaded here. perhaps give the resource a different name so that it's clear that it shouldn't just be the constant "image".

Reply:
done